tests/Unit/PointwiseFunctions/GeneralRelativity/Test_Ricci.cpp

```diff
@@ -21,39 +21,52 @@
 // IWYU pragma: no_include <boost/preprocessor/tuple/reverse.hpp>
 
 namespace {
-template <size_t Dim, IndexType TypeOfIndex, typename DataType>
+template <size_t Dim, typename DataType>
 void test_compute_item_in_databox(const DataType& used_for_size) {
   TestHelpers::db::test_compute_tag<
-      gr::Tags::SpatialRicciCompute<3, Frame::Inertial, DataType>>(
+      gr::Tags::SpatialRicciCompute<Dim, Frame::Inertial, DataType>>(
       "SpatialRicci");
+  TestHelpers::db::test_compute_tag<
+      gr::Tags::SpatialRicciScalarCompute<Dim, Frame::Inertial, DataType>>(
+      "SpatialRicciScalar");
 
   MAKE_GENERATOR(generator);
   std::uniform_real_distribution<> distribution(-3.0, 3.0);
   const auto nn_generator = make_not_null(&generator);
   const auto nn_distribution = make_not_null(&distribution);
 
   const auto christoffel_2nd_kind = make_with_random_values<
-      tnsr::Abb<DataType, Dim, Frame::Inertial, TypeOfIndex>>(
+      tnsr::Abb<DataType, Dim, Frame::Inertial, IndexType::Spatial>>(
```

Comment:
Does this mean the test fails for spacetime tensors? If so, please generalize so the code works for spacetime as well as spatial tensors, and then don't assume spatial tensors here

Reply:
I forgot to take those restrictions out during my testing of the scalar alone, just pushed a fixup with tests passing for both spacetime and spatial